packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/search/components/field_query_parser.py

```
@@ -0,0 +1,205 @@
+    field_queries: list[FieldQuery]
+    text_query: str
+    original_query: str
+
+
+class FieldQueryParser:
+    """Parses field-specific query syntax and converts to Qdrant filters."""
+    
+    # Supported field mappings (field_name -> qdrant_payload_key)
+    SUPPORTED_FIELDS = {
+        "document_id": "document_id",
+        "source_type": "source_type", 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)